### DISCUSSION TITLE
WiFi signal strength mapping  for heatmap

### DISCUSSION THREAD

**Discussion by MiBu-dev** [Tasshack/dreame-vacuum#524] [Ideas]
Hi 
Just a fought ... In Diagnostic sensors there is just MAC - but there is no WiFi signal strength info - for once if location for base is chosen adequately :) but putting together Signal strength with location on MAP would make neat WiFi Heat map coverage... (there are apps for phones - but one has to make image of theirs flat :)  - robot could do this automatically when scouting 

I found article that there is possible to upload python script to do that - however they used JTAG to  access local file system :(

maybe signal strength can be get somehow ? I routinelly add that for my ESPHome devices  to diagnose if location requires  modding for external antenna.

Great Integration!

**Comment by Tasshack** [comment]
I can add wifi info data from the vacuum but the problem is App does not show that particular setting item to shared accounts and I don't own a vacuum that can wifi map.

**Comment by MiBu-dev** [comment]
I thought rather  of making it as picture glance   - lidar map with overlayed   heat map  - so it is done in HA rather than on vac's cloud :)
